packages/workspace/src/utilities/strip-source-code.ts

```diff
@@ -4,17 +4,39 @@ export function stripSourceCode(scanner: Scanner, contents: string): string {
   if (contents.indexOf('loadChildren') > -1) {
     return contents;
   }
-  if (contents.indexOf('require') > -1) {
-    return contents;
-  }
 
   scanner.setText(contents);
   let token = scanner.scan();
   const statements = [];
   let start = null;
+  let ignoreNextLine = false;
   while (token !== SyntaxKind.EndOfFileToken) {
     const potentialStart = scanner.getStartPos();
     switch (token) {
+      case SyntaxKind.MultiLineCommentTrivia:
+      case SyntaxKind.SingleLineCommentTrivia: {
+        const isMultiLineCommentTrivia =
+          token === SyntaxKind.MultiLineCommentTrivia;
+        const start = potentialStart + 2;
+        token = scanner.scan();
+        const end = scanner.getStartPos() - (isMultiLineCommentTrivia ? 2 : 0);
+        const comment = contents.substring(start, end).trim();
+        if (comment === 'nx-ignore-next-line') {
+          while (
+            token === SyntaxKind.WhitespaceTrivia ||
+            token === SyntaxKind.NewLineTrivia
+          ) {
+            token = scanner.scan();
+          }
+          ignoreNextLine =
```

Comment:
I think we should ignore the next line regardless of what's in it?

I believe this doesn't get ignored at the moment..

```ts
// nx-ignore-next-line
const fs = require('fs');
```

Maybe keep scanning until 2 new lines are hit. 1 for the new line after the comment and 1 for the one at the end of the next line.

Reply:
Addressed it